src/navigation/NavBar.js

```diff
@@ -73,6 +74,9 @@ export const NavBar = ({ t }) => {
         </div>
       </div>
       <div className='dn db-l navbar-footer mb2 tc center f7 o-80 glow'>
+        <div className='mb4'>
+          <ThemeToggle />
+        </div>
```

Comment:
:thought_balloon:  spale in left mavbar is limited especially in ipfs-desktop where UI acts as native app.  

perhaps it is  better to move this toggle to top right corner, where we already have two items:

> ![image](https://github.com/user-attachments/assets/7a1f407b-29df-4fe4-bb90-83eb5359ff76)

Reply:
oh, yeah! that sounds goood, @lidel! 

i originally thought of placing it there, but couldn't locate the icons in the code. I'll try looking for them and update the theme toggle's position